moveit_ros/moveit_servo/config/panda_simulated_config.yaml

```diff
@@ -53,6 +53,7 @@ num_outgoing_halt_msgs_to_publish: 4
 
 ## Configure handling of singularities and joint limits
 lower_singularity_threshold:  17.0  # Start decelerating when the condition number hits this (close to singularity)
+approaching_stop_singularity_threshold: 23.5 # Stop when the condition number hits this and we are continuing to approach singularity
 hard_stop_singularity_threshold: 30.0 # Stop when the condition number hits this
```

Comment:
```suggestion
hard_stop_singularity_threshold: 30.0 # Stop when the condition number hits this and we are moving away from singularity
```

Reply:
My understanding is, `hard_stop_singularity_threshold` should always be greater than `approaching _stop_singularity_threshold` or this strategy doesn't make sense. I'll make a suggestion below to verify that when the parameters are loaded